src/load/azext_load/operations/update.py

```diff
@@ -0,0 +1,50 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#
+# --------------------------------------------------------------------------------------------
+
+from azure.cli.core.aaz import has_value
+from azure.cli.core.azclierror import (
+    InvalidArgumentValueError
+)
+
+from msrestazure.tools import is_valid_resource_id
+
+from ..aaz.latest.load._update import Update
+
+
+class LoadTestUpdate(Update):
+
+    def _cli_arguments_loader(self):
+        args = super()._cli_arguments_loader()
+
+        # encryption_identity_type args are not exposed
+        # encryption_identity_type is populated based on encryption_identity arg value
+        args = [(name, arg) for (name, arg) in args if name not in ["encryption_identity_type"]]
+        return args
+
+    def pre_operations(self):
+        args = self.ctx.args
+        identity_type_str = str(args.identity_type)
```

Comment:
```suggestion
        identity_type_str = args.identity_type.to_serialized_data()
```

Reply:
When you use .to_serialized_data() of AAZStringArg, the return value type will be string, None (if argument is nullable and user pass `null`) or AAZUndefined (not has value).

Reply:
Updated it, as recommended